controllers/space/nstemplatetier_spaces_mapper_test.go

```diff
@@ -0,0 +1,103 @@
+package space_test
+
+import (
+	"testing"
+
+	tierutil "github.com/codeready-toolchain/host-operator/controllers/nstemplatetier/util"
+	"github.com/codeready-toolchain/host-operator/controllers/space"
+	"github.com/codeready-toolchain/host-operator/pkg/apis"
+	tiertest "github.com/codeready-toolchain/host-operator/test/nstemplatetier"
+	spacetest "github.com/codeready-toolchain/host-operator/test/space"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func TestMapNSTemplateTierToSpaces(t *testing.T) {
+
+	// given
+	logf.SetLogger(zap.New(zap.UseDevMode(true)))
+	err := apis.AddToScheme(scheme.Scheme)
+	require.NoError(t, err)
+
+	t.Run("single match", func(t *testing.T) {
+		// given
+		nsTmplTier := tiertest.BasicTier(t, tiertest.CurrentBasicTemplates)
```

Comment:
let's skip such details, shall we?

Reply:
sure, no problem at all :wink: . I just thought that I could suggest it since you will be changing the code anyway and it could make the code two lines shorter :sweat_smile: